lib/govobject/types/trigger.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+var _ = require('lodash');
+var $ = require('../../util/preconditions');
+var GovObject = require('../govobject');
+var errors = require('../../errors');
+var inherits = require('util').inherits;
+
+/**
+ * Represents 'Trigger' Governance Object
+ *
+ * @constructor
+ */
+function Trigger(serialized) {
+  GovObject.call(this, serialized);
+}
+
+inherits(Trigger, GovObject);
+
+Trigger.prototype.dataHex = function() {
+
+  var _govObj = {
+    event_block_height: this.event_block_height,
+    payment_addresses: this.payment_addresses,
+    payment_amounts: this.payment_amounts,
+    proposal_hashes: this.proposal_hashes,
+    type: this.type,
+  };
+  // screwy data shims 'til we can fix this on dashd
+  var inner = ['trigger', _govObj];
+  var outer = [inner];
+
+  //Done to match the rpc returns from dashd todo: follow up, if bug or not.
+  var strigified = JSON
+    .stringify(outer)
+    .replace(/,/g,', ')
+    .replace(/:/g,': ');
+
+  return strigified
+};
+
+Trigger.prototype._newGovObject = function() {
+  this.event_block_height = "";
```

Comment:
This should be an integer, not sure how JS handles this (I see you have something about "number" below).